Sources/AZPolicyFixtures/PolicyType.swift

```diff
@@ -0,0 +1,194 @@
+// PolicyType.swift
+// azure-devops-swift
+//
+// Copyright © 2023 MFB Technologies, Inc. All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+import AZCore
+import AZCoreFixtures
+import AZPolicy
+import Foundation
+
+// swiftlint:disable line_length
+extension AZFixtures.PolicyType {
+    public static let secretsScanningRestriction = AZPolicyType(
+        id: "ec003f37-8db0-4e10-992a-a2895045752c",
```

Comment:
I'm really unfamiliar with the Azure DevOps, but I noticed quite a few floating IDs here. Are these safe to expose in an OSS project?

Reply:
Those individual resource IDs should not be sensitive. I did make sure to scrub all references to any real organization or project to be extra safe.